src/store/calendars.js

```diff
@@ -336,7 +338,10 @@ const actions = {
 					return mapDavCollectionToCalendar(calendar)
 				})
 			})
-
+		
+		// remove calendars which don't support tasks
+		calendars = calendars.filter(calendar => calendar.supportsTasks)
```

Comment:
@skjnldsv @georgehrke Is this a good solution to filter VTODO calendars or should this be in the CDAV library?

Reply:
@raimund-schluessler filter can be crazy expensive on ressources.
I'll suggest you make performances tests to ensure this is not a bottleneck for your app :)

Reply:
Well, this should only be called once on app load. But I am open to other solutions ;)

Reply:
Looking at the code, the objects are quite simple, so it should be fine, can you do a performance tests before and after the filter to check?

Reply:
I don't have much experience with performance testing. Should I just do a `console.time('Filter') ... console.timeEnd('Filter')` or what do you recommend?

Reply:
Takes 1ms for five calendars: `calendars.filter: 1ms`
This seems to be ok for the initial loading time. I will check the other places where I use `filter` as well.

Reply:
Ahaha, perfect :p